### PR TITLE
Skip copied entities to generate script

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/3488

## Checklist
- [x] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [x] Added tests

### CODE REVIEW

File: compiler-plugin-test/src/test/resources/test-src/plugin/package_20/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+version = "0.1.0"
+
+[build-options]
+observabilityIncluded = true
+
+[[dependency]]
+org="ballerina"
+ name="persist"
+ repository="local"
+ version="0.1.0"
```
Comment (daneshk) [inline compiler-plugin-test/src/test/resources/test-src/plugin/package_20/Ballerina.toml:13]:
```suggestion
[[dependency]]
org="ballerina"
name="persist"
repository="local"
version="0.1.0"
```

Comment (daneshk) [reply compiler-plugin-test/src/test/resources/test-src/plugin/package_20/Ballerina.toml:13]: Do we need this

**Review by daneshk — APPROVED** [review]
Add a minor comment. Overall LGTM

File: compiler-plugin-test/src/test/resources/test-src/plugin/package_20/Ballerina.toml (hunk 1 above)
Comment (kalaiyarasiganeshalingam) [reply compiler-plugin-test/src/test/resources/test-src/plugin/package_20/Ballerina.toml]: No need, Added to test local.

**Review by daneshk — APPROVED** [review]

**Pull request merged**